src/Fixer/Phpdoc/PhpdocOrderByValueFixer.php

```diff
@@ -121,6 +121,13 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens)
                     $type
                 );
 
+                if (\in_array($type, ['property', 'property-read', 'property-write'], true)) {
```

Comment:
Thinking about adding support for `@method` annotations as well, what do you think?